src/mlpack/methods/sparse_svm/sparse_svm.hpp

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file sparse_svm.hpp
+ * @author Ayush Chamoli
+ *
+ * An implementation of Sparse SVM.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_SPARSE_SVM_SPARSE_SVM_HPP
+#define MLPACK_METHODS_SPARSE_SVM_SPARSE_SVM_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <ensmallen.hpp>
+
+#include "sparse_svm_function.hpp"
+
+namespace mlpack {
+namespace svm {
+
+class SparseSVM
```

Comment:
Can we add some class documentation here describing what a sparse SVM is (briefly), any relevant paper references, and maybe point to where more information can be found?